plenty_api/constants.py

```diff
@@ -116,6 +118,13 @@
     ],
     'property': [
         'propertyId'
+    ],
+    'bi_raw': [
+        'dataName',
+        'processStatus',
+        'createdAtTimestamp',
+        'sortOrder',
+        'itemsPerPage,'
```

Comment:
Also, the REST API actually provides a `page` parameter. But I know this isn't very practical to use here so I thought about. I think we could implement a general `pages` refine parameter for all get requests like:
[within **__repeat_get_request_for_all_records**]
```python
...
         page_info = utils.sniff_response_format(response=response, query=query)
+        # Handle page slices (custom selection of pages)
+        slice_start = ''
+        slice_end = ''
+        if 'pages' in query:
+            pages = query['pages']
+            slice_start = pages['start_page'] if 'start_page' in pages else ''
+            slice_end = pages['end_page'] if 'end_page' in pages else ''
+            logging.debug(f"Using page slice [{slice_start}:{slice_end}]")
+        if slice_start and slice_start >= 2:
+            entries = []
+        else:
+            entries = response[page_info['data']]
...
+            if slice_start and page < slice_start:
+                # Skip pages before the beginning of the slice
+                continue
             entries += response[page_info['data']]
 ...
+            if slice_end and page == slice_end:
+                # Skip requests for pages after the end of the slice
+                break
```

This would enable the user to take a specific sub set (like for example the first 300 items).